webapp/src/app/season-selector/season-selector.component.ts

```diff
@@ -27,7 +27,7 @@ export class SeasonSelectorComponent implements OnInit {
 
         if (!this.championshipService.getCurrentChampionship()) {
             this.championshipService.getCurrent().subscribe((championships) => {
-                if (championships.length === 0) {
+                if (championships !== null && championships.length === 0) {
```

Comment:
I'd write `(championships != null && !championship.length)` :)

Reply:
please, merge as it is, I have another PR to put all "null" together :)